app/views.py

```diff
@@ -0,0 +1,30 @@
+from app import app
+from flask import render_template
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/<input>')
+def print_user_input(input):
+    if input.__contains__('='):
```

Comment:
http://www.nooooooooooooooo.com/

__contains__ is part of the meta object protocol, you *never* use it directly. this function is being invoked by `in`, i.e. your code is equivalent to `if '=' in input:`  … do tell me this reads better :)